index.js

```diff
@@ -273,7 +273,7 @@ export default class Emittery {
 		}
 	}
 
-	on(eventNames, listener) {
+	on(eventNames, listener, options) {
```

Comment:
```suggestion
	on(eventNames, listener, {signal} = {}) {
```

to simplify the code a bit.